core/src/main/java/hudson/scm/ChangeLogParser.java

```diff
@@ -52,6 +63,16 @@
         }
     }
 
+    /**
+     * Method for compatibility with AbstractBuild.
```

Comment:
I would just say, `@deprecated this method should not be used directly anymore, use the {@link ...} method instead`. Should not override annotation may be helpful as well (`@Restricted(ProtectedExternally.class) `)